Cannot find module 'babel-runtime/helpers/classCallCheck'

After fresh install.
I got an error that babel-runtime is not installed.

```
module.js:457
    throw err;
    ^

Error: Cannot find module 'babel-runtime/helpers/classCallCheck'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\src\node_modules\rally\dist\restapi.js:7:24)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
```

Works in case babel-runtime installed either globally or locally.

--- a/.babelrc
+++ b/.babelrc
@@ -1,10 +1,9 @@
 {
   "plugins": [
-    "transform-runtime",
-    ["transform-es2015-classes"]
+    "transform-runtime"
   ],
   "presets" : [
-    "es2015",
+    "latest",
     "stage-1"
   ],
   "env": {

--- a/package.json
+++ b/package.json
@@ -34,12 +34,13 @@
     "request": "2.74.0"
   },
   "devDependencies": {
-    "babel-cli": "6.11.4",
+    "babel-cli": "6.14.0",
     "babel-eslint": "6.1.2",
     "babel-plugin-add-module-exports": "0.2.1",
-    "babel-plugin-transform-runtime": "6.9.0",
-    "babel-preset-es2015": "6.9.0",
-    "babel-preset-stage-1": "6.5.0",
+    "babel-plugin-transform-runtime": "6.15.0",
+    "babel-preset-latest": "6.14.0",
+    "babel-preset-stage-1": "6.13.0",
+    "babel-register": "6.14.0",
     "babel-runtime": "6.11.6",
     "cross-env": "2.0.0",
     "eslint": "2.13.1",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "rally",
-  "version": "2.1.0",
+  "version": "2.1.1",
   "description": "Rally REST Toolkit for Node.js",
   "contributors": [
     {

